app/views.py

```diff
@@ -0,0 +1,30 @@
+from app import mulungwishi_app as url
+from flask import render_template
+
+
+@url.route('/')
+def index():
+    return render_template('index.html')
+
+
+@url.route('/<query>')
+def print_user_input(query):
+    if '=' in query:
+        query_container, query_value = query.split('=')
+        return 'Your query is {} which is equal to {}'.format(query_container, query_value)
```

Comment:
want to do something cool?

```
return 'Your query is {} which is equal to {}'.format(*query.split('='))
```

embrace the power of python @admiral96 ! :)

(also, that pair is usually called a key-value-pair :)